docs/docs/guides/core-types/media.md

```diff
@@ -171,3 +171,118 @@ This audio is logged to Weave and automatically displayed in the UI, along with
 :::tip
 Try our cookbook for [Audio Logging](/reference/gen_notebooks/audio_with_weave) or <a href="https://colab.research.google.com/github/wandb/weave/blob/master/docs/./notebooks/audio_with_weave.ipynb" target="_blank" rel="noopener noreferrer" class="navbar__item navbar__link button button--secondary button--med margin-right--sm notebook-cta-button"><div><img src="https://upload.wikimedia.org/wikipedia/commons/archive/d/d0/20221103151430%21Google_Colaboratory_SVG_Logo.svg" alt="Open In Colab" height="20px" /><div>Open in Colab</div></div></a>. The cookbook also includes an advanced example of a Real Time Audio API based assistant integrated with Weave.
 :::
+
+## Files (PDFs, CSVs)
+
+<Tabs groupId="programming-language" queryString>
+  <TabItem value="python" label="Python" default>
+  
+  ### PDFs
+  You can log and visualize various file types like [PDFs](#pdfs) and [CSVs](#csvs) using the `weave.File` object. This includes PDFs with varying lengths or content. Weave captures file metadata like path, MIME type, and size in traces. You can also [preview](#preview-a-file) or [download](#download-a-file) traced files from the **Traces** tab.
```

Comment:
You can only preview a `weave.File` if we have built a viewer for that mime type, currently just PDFs.